src/main/scala/org/allenai/plugins/CoreRepositories.scala

```diff
@@ -61,5 +67,15 @@ object CoreRepositories {
         if (isSnapshot.value) Some(ai2PublicSnapshots) else Some(ai2PublicReleases)
       }
     }
+
+    val ai2BintrayPrivate = Seq(
```

Comment:
These work in isolation, as long as a license is set.  Projects can add further settings on top of these too.